packages/vuetify/src/composables/size.ts

```diff
@@ -2,33 +2,28 @@
 import { computed } from 'vue'
 import propsFactory from '@/util/propsFactory'
 
-// Types
-import type { Prop } from 'vue'
-
-const allowedSizes = ['x-small', 'small', 'default', 'large', 'x-large'] as const
-
-type Size = typeof allowedSizes[number]
+const predefinedSizes = ['x-small', 'small', 'default', 'large', 'x-large']
```

Comment:
Should this be capitalized, similar to a global variable? I feel like this would be used in more than one place.